quipucords/scanner/network/connect.py

```diff
@@ -162,11 +162,12 @@ def run_with_result_store(self, result_store):
         serializer = SourceSerializer(self.scan_task.source)
         source = serializer.data
 
-        if self.scan_job.options is None:
-            forks = ScanOptions.get_default_forks()
-        else:
+        if self.scan_job.options is not None:
             forks = self.scan_job.options.max_concurrency
+        else:
+            forks = ScanOptions.get_default_forks()
         connection_port = source['port']
+        use_paramiko = source['use_paramiko']
```

Comment:
Does this throw an error if `use_paramiko` is not specified when creating the source?

Reply:
Setting should be handled if options is None.